dagshub/upload/wrapper.py

```diff
@@ -261,10 +386,33 @@ def add_dir(self, local_path, glob_exclude=""):
 
     @staticmethod
     def _clean_directory_name(directory: str):
+        """
+        The _clean_directory_name function takes a directory name as an argument and returns the normalized path of that directory.
+        For example, if the input is ../../../ then it will return /. If the input is ./ then it will return ..
+        If there are any other characters in the string, they will be ignored and only alphanumeric characters (a-zA-Z0-9) 
+        will be kept.
+        
+        :param directory (str): Specify the directory that will be cleaned
+        :return: The normalized path of the directory(The directory name with the path separator normalized to a forward slash)
+        
+        """
+       
         return os.path.normpath(directory)
 
     @staticmethod
     def get_file(file: Union[str, IOBase], path=None):
+        """
+        The get_file function is a helper function that takes in either a string or an IOBase object and returns
+        a tuple containing the file's name and the file itself. If no path is provided, it will default to the name of
+        the file. This function also handles exceptions such as when you try to pass in a directory instead of a file.
```

Comment:
Redundant in my opinion...
```suggestion
        the file.
```